### ISSUE TITLE
Upgrading to 1.3.1 in (newly upgraded) Ember-CLI 2.3.0 project fails

### ISSUE THREAD

**Issue by patcoll** [kellyselden/broccoli-jscs#68]
Started getting the error below when we upgraded to 1.3.1. Pinning to 1.3.0 appears to fix symptom.

Found these references but nothing helped fix the problem:

- https://github.com/jscs-dev/node-jscs/issues/1567
- https://github.com/jscs-dev/esprima-harmony/issues/3

The following is the full error (replaced full path with [...] because paranoia):

    version: 2.3.0
    Build failed.
    File: assets/frontend.js
    Unable to add an error, `line` should be a number greater than 0 but undefined given
    AssertionError: Unable to add an error, `line` should be a number greater than 0 but undefined given
        at Object.Errors._validateInput ([...]/frontend/node_modules/broccoli-jscs/node_modules/jscs/lib/errors.js:69:9)
        at Object.Errors.add ([...]/frontend/node_modules/broccoli-jscs/node_modules/jscs/lib/errors.js:46:14)
        at StringChecker._addParseError ([...]/frontend/node_modules/broccoli-jscs/node_modules/jscs/lib/string-checker.js:211:16)
        at null.<anonymous> ([...]/frontend/node_modules/broccoli-jscs/node_modules/jscs/lib/string-checker.js:97:22)
        at Array.forEach (native)
        at StringChecker.checkString ([...]/frontend/node_modules/broccoli-jscs/node_modules/jscs/lib/string-checker.js:95:31)
        at JSCSFilter.processString ([...]/frontend/node_modules/broccoli-jscs/index.js:91:31)
        at [...]/frontend/node_modules/broccoli-jscs/node_modules/broccoli-persistent-filter/lib/strategies/persistent.js:35:42
        at lib$rsvp$$internal$$tryCatch ([...]/frontend/node_modules/broccoli-jscs/node_modules/broccoli-persistent-filter/node_modules/rsvp/dist/rsvp.js:1036:16)
        at lib$rsvp$$internal$$invokeCallback ([...]/frontend/node_modules/broccoli-jscs/node_modules/broccoli-persistent-filter/node_modules/rsvp/dist/rsvp.js:1048:17)
        at lib$rsvp$$internal$$publish ([...]/frontend/node_modules/broccoli-jscs/node_modules/broccoli-persistent-filter/node_modules/rsvp/dist/rsvp.js:1019:11)
        at lib$rsvp$asap$$flush ([...]/frontend/node_modules/broccoli-jscs/node_modules/broccoli-persistent-filter/node_modules/rsvp/dist/rsvp.js:1198:9)
        at nextTickCallbackWith0Args (node.js:419:9)
        at process._tickCallback (node.js:348:13)

**Comment by kellyselden** [comment]
cc @Turbo87

**Comment by Turbo87** [comment]
@patcoll do you have a repo to reproduce this problem?

**Comment by patcoll** [comment]
https://github.com/luma-institute/ember-230-testing

I believe I've answered my own question, but I welcome feedback.

I was following both the general "Usage" instructions and the Ember-CLI instructions. Apparently I only needed to follow the Ember-CLI instructions. Once I made the following change:

https://github.com/luma-institute/ember-230-testing/commit/28668476d5b97d658ed3d8145a770ea29cce82e8

The reported issue went away.

Am I thinking about this in the right way? Was I double-initializing broccoli-jscs somehow and now I'm not? I'm definitely a novice in both Broccoli and JSCS so I welcome any feedback or direction.

Thanks!
